catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -13,11 +13,10 @@
  */
 package org.codice.ddf.catalog.ui.query;
 
-import static spark.Spark.after;
-import static spark.Spark.exception;
-import static spark.Spark.post;
+import static spark.Spark.*;
```

Comment:
I think intelliJ snuck this in on me, but will be fixing the formatting issues presently now that I've imported the formatted .jar. :+1: